tactical-microgrid-standard/controller/Controller.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CONTROLLER_CONTROLLER_H
+#define CONTROLLER_CONTROLLER_H
+
+#include "Common.h"
+#include "common/Handshaking.h"
+
+class Controller : public Handshaking {
```

Comment:
Do we have to reconcile this with https://github.com/OpenDDS/DeveloperResources/blob/main/tactical-microgrid-standard/common/Controller.h?

Reply:
Yes, I think it makes sense to merge them.